Topic5/Topic5.sql

```diff
@@ -0,0 +1,80 @@
+CREATE DATABASE high_school;
+
+USE high_school;
+
+CREATE TABLE Student(
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	registration_number INT UNSIGNED NOT NULL,
+	date_of_birth DATE NOT NULL,
+	
+	INDEX(first_name,last_name),
+	
+	CONSTRAINT pk_student PRIMARY KEY (registration_number)
+	) ENGINE=InnoDB;
+	
+CREATE TABLE Teacher(
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	file_number INT UNSIGNED NOT NULL,
+	date_of_birth DATE NOT NULL,
+	
+	CONSTRAINT pk_teacher PRIMARY KEY (file_number)
+	)ENGINE=InnoDB;
+	
+CREATE TABLE Course(
+	name_course VARCHAR(20) NOT NULL,
+	id_course INT UNSIGNED NOT NULL,

```

Comment:
This field could be an autoincremental to avoid the need to define it on an Insert clause